nodejs/awsx/apigateway/README.md

```diff
@@ -0,0 +1,139 @@
+# Pulumi API Gateway Components
+
+Pulumi's API for simplifying working with [API Gateway](https://aws.amazon.com/api-gateway/). The API currently provides ways to define a route that accepts and forwards traffic to a specified destination.
+
+## Defining an Endpoint
+
+To define an endpoint you will either need to specify the routes or a swagger string. You can also define the stage name (else it will default to "stage"). A `stage` is an addressable instance of the Rest API.
+
```

Comment:
i would not even cover the swagger part TBH.  It's not really an expected way we think people will use.  I would also lead with the event handler stuff.